phpstan-baseline.neon

```diff
@@ -339,7 +339,10 @@ parameters:
 			message: "#^Unreachable statement \\- code above always terminates\\.$#"
 			count: 1
 			path: lib/Security/Encoder/AbstractUserAwarePasswordEncoder.php
-
+		-
+            message: "#^Call to an undefined method Symfony\\\\Component\\\\Security\\\\Core\\\\User\\\\UserInterface\\:\\:getUsername\\(\\)\\.$#"
```

Comment:
please revert these changes and merge master. thanks!

Reply:
done